src/khoj/interface/web/chat.html

```diff
@@ -281,10 +282,17 @@
                 </a>
             {% endif %}
             <nav class="khoj-nav">
-                <a class="khoj-nav khoj-nav-selected" href="/chat">Chat</a>
-                <a class="khoj-nav" href="/">Search</a>
+                <a class="khoj-nav khoj-nav-selected" href="/chat">💬 Chat</a>
+                <a class="khoj-nav" href="/">🔎 Search</a>
                 {% if not demo %}
-                    <a class="khoj-nav" href="/config">Settings</a>
+                    <!-- Dropdown Menu -->
```

Comment:
Could it be worthwhile to use Jinja Macros to define this account component? That way we won't have to copy/paste as much and we'll get better re-use. https://jinja.palletsprojects.com/en/3.0.x/templates/#macros

Reply:
Great idea, didn't know that existed. Reduced duplication of HTML by creating a `utils.html` with the macro to generate the navigation pane